components/ContentCard.vue

```diff
@@ -1,13 +1,16 @@
 <template>
-  <li class="card">
+  <b-card
+    :title="cardTitle"
+    :img-src="imageUrl"
+    img-alt="This is an image"
+    img-top
+    footer="Creator of this object"
+    class="mb-4">
```

Comment:
when running via 'npm run dev' this line throws a linting error, it expects an additional linebreak.